src/aux-server/aux-web/shared/scene/Game.ts

```diff
@@ -863,13 +869,17 @@ export abstract class Game {
         // Main scene camera.
         this.setCameraType('orthographic');
 
+        const defaultLighting = true;
```

Comment:
@TroyceGowdy Can you remove this line? It looks like this `defaultLighting` variable was left over from some testing